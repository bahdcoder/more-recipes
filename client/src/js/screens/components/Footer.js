import React from 'react';

export default class Footer extends React.Component {
  render() {
    return (
      <footer className="footer">
        <p>More-recipes app by bahdcoder @andela</p>
      </footer>
    );
  }
}
