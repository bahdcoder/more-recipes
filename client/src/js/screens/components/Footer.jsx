import React from 'react';

const Footer = () => ((
  <footer className="footer mt-5">
    <p>More-recipes app by bahdcoder @andela</p>
  </footer>
));

export default Footer;
