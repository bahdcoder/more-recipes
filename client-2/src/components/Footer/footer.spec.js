/* global expect */
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

it('The footer renders without crashing', () => {
  const tree = renderer.create((
    <Footer />
  )).toJSON();
  expect(tree).toMatchSnapshot();
});
