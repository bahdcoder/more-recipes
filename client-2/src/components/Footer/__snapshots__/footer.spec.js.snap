// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The footer renders without crashing 1`] = `
<footer
  className="footer mt-5"
>
  <p>
    More-recipes app by bahdcoder @andela
  </p>
</footer>
`;
